Sources/FSCameraView.swift

```diff
@@ -84,7 +84,7 @@ final class FSCameraView: UIView, UIGestureRecognizerDelegate {
             
             // Focus View
             self.focusView         = UIView(frame: CGRect(x: 0, y: 0, width: 90, height: 90))
-            let tapRecognizer      = UITapGestureRecognizer(target: self, action: Selector("focus:"))
+            let tapRecognizer      = UITapGestureRecognizer(target: self, action:Selector("focus:"))
```

Comment:
In Swift 2.1 could write either "focus:" or Selector("focus:"). If I'm not wrong.